test/plugin_reload_test.go

```diff
@@ -0,0 +1,103 @@
+package test
+
+import (
+	"bytes"
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/mholt/caddy"
+)
+
+func TestPluginReload(t *testing.T) {
```

Comment:
Nit: drop `Plugin` here.

Reply:
There's another fn with the name `TestReload` and hence I added the work `Plugin`.

Reply:
ah, yeah, we're in the global test/ dir; my bad.